solc/src/lib.rs

```diff
@@ -3,7 +3,7 @@ mod platform {
 	use std::process::Command;
 
 	pub fn solc() -> Command {
-        Command::new("solcjs")
+        Command::new("solc")
```

Comment:
What's the reason behind this change? Currently we only require `npm install -g solc`, which installs `solcjs`. Do we need to use CPP version for some reason?

Reply:
While I agree with you that compiler change shouldn't be included in this PR, the general idea is to move to `solc` — I personally have much shorter compile times with it. (Eventually, we want to get rid of a separate compiler requirement at all, which is tracked in #8 )

Reply:
I agree that #8 is the best option, but I don't think it's going to happen soon. For me currently compile time of contracts is negligible compare to compilation time of all solaris dependencies. Also `solcjs` is way easier to install, which is pretty important for later adoption.

I would be in favour of detecting which of the two is available (with `solc` being prefered) and using that one, but indeed should be part of separate PR.

Reply:
> I would be in favour of detecting which of the two is available (with solc being prefered) and using that one, but indeed should be part of separate PR.

This compiler change got rolled into the commits with this branch by mistake. I have another branch (lght-solc) which tracks these changes. I can make a PR for that branch, and attempt some detection of what is available on the system.

> I personally have much shorter compile times with it.

^ same for me, and is my main motivation for moving away from solcjs. Because of the current CI setup (and proper separation of concerns), I will revert the compiler changes in this PR.